include/RAJA/util/resource.hpp

```diff
@@ -70,6 +70,16 @@ namespace RAJA
     using type = camp::resources::Cuda;
   };
 
+  template <bool Async, int num_threads>
+  struct get_resource<cuda_launch_t<Async, num_threads>>{
+    using type = camp::resources::Cuda;
+  };
+
+  template <bool Async, int num_threads, size_t BLOCKS_PER_SM>
+  struct get_resource<RAJA::policy::cuda::cuda_launch_explicit_t<Async, num_threads, BLOCKS_PER_SM>>{
```

Comment:
I noticed this policy `RAJA::policy::cuda::cuda_launch_explicit_t` is tucked away in the namespace do we want to expose this under the `RAJA` namespace.

Reply:
This was hidden because it's not a common use case, and the corresponding `hip_launch_explicit_t` was not completely implemented due to their interface being quirky. I think it's better to keep it tucked away for now, and maybe expose it if it becomes more popular?

Reply:
Perhaps, I'm not familiar when using this would help code's performance. If there is some insight it might be good to document the use case so users could integrate accordingly.

Reply:
FWIW, I know of only one case where a user being able to provide min blocks per sm made a performance difference, and I'm not sure why it makes a difference. It was originally done for one specific RAJA kernel launch method to address that user issue. Later it was filled in for all other kernel launch methods, as appropriate, for consistency. It may be a good idea at some point when someone has some free time to try different values for a few kernels to see what the impact is.